restapi/provider.go

```diff
@@ -179,6 +210,19 @@ func configureProvider(d *schema.ResourceData) (interface{}, error) {
 	if v, ok := d.GetOk("destroy_method"); ok {
 		opt.destroy_method = v.(string)
 	}
+	if _, ok := d.GetOk("oauth_client_credentials"); ok {
```

Comment:
These three lines look like a duplicate of the first few lines in the next check

Reply:
whoops yes something went wrong there :)